components/Banner/examples/BannerTypicalUseExampleViewController.m

```diff
@@ -129,230 +130,238 @@ + (NSDictionary *)catalogMetadata {
   };
 }
 
+#pragma mark - UIViewController
+
+- (void)viewDidLayoutSubviews {
+  [super viewDidLayoutSubviews];
+
+  self.contentView.frame = self.view.bounds;
+  self.contentViewLabel.center = self.contentView.center;
+  CGRect exampleListTableViewFrame =
+      CGRectMake(0, self.view.bounds.size.height - exampleListTableViewHeight,
+                 self.view.bounds.size.width, exampleListTableViewHeight);
+  self.exampleListTableView.frame = exampleListTableViewFrame;
+
+
+  CGSize bannerViewSize =
+  [self.bannerView sizeThatFits:CGSizeMake(CGRectGetWidth(self.view.bounds), CGFLOAT_MAX)];
```

Comment:
Indent this line.

Reply:
Good catch. Addressed in 725ab5985288813e9e77af78e1dc1e437128da90